### DISCUSSION TITLE
Use 'getdate()' inside ListRequest.Criteria to call List endpoint

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#5898] [Help] [answered]
Hey, I am trying to fetch a list from my XYZGrid.ts. I want to compare the DueDate property with current date.

```
Criteria: Serenity.Criteria.and(
                    [[XYZRow.Fields.Username], '=', Authorization.userDefinition.Username],
                    [[XYZRow.Fields.DueDate], '<=', 'GETDATE()']
                )
```
I am getting following exception: **Microsoft.Data.SqlClient.SqlException: 'Conversion failed when converting date and/or time from character string.'**

Any ideas? Thanks

**Comment by reach2rv** [comment, marked as answer]
You should use new Date method of javascript it will pass current date to the criteria

**Reply by ghost** [reply]
That was fast, worked like charm. Thanks!
